### DISCUSSION TITLE
RF accompanying slice select gradients are bound to z channel.

### DISCUSSION THREAD

**Discussion by h3lg3** [imr-framework/pypulseq#225] [Q&A] [answered]
Hello again,

I realized that the gradients returned by `make_adiabatic_pulse()` or `make_sinc_pulse()` are by default assigned to the z-channel. For example `line 148` in `make_sinc_pulse()`. Should the channel become an optional input argument of make_sinc_pulse()? 

Thanks for your opinion.
Kind regards
Helge

In make_sinc_pulse():
```
    gz = make_trapezoid(
        channel="z", system=system, flat_time=duration, flat_area=area
    )
```

**Comment by schuenke** [comment, marked as answer]
hmmm, the gradients returned with the RF pulse are for slice selection and the slice direction is by convention the z-axis. So in my opinion specifying a channel does not make sense. Also keep in mind that we would have to rename `return_gz` to something like `return_grad`, which would break compatibility with current python scripts. Not just the ones in the PyPulseq repo, but all scripts from all users.
